src/3d/terrain/qgsterrainentity_p.cpp

```diff
@@ -58,7 +58,7 @@ class TerrainMapUpdateJobFactory : public QgsChunkQueueJobFactory
 
 
 QgsTerrainEntity::QgsTerrainEntity( const Qgs3DMapSettings &map, Qt3DCore::QNode *parent )
-  : QgsChunkedEntity( map.maxTerrainScreenError(), map.terrainGenerator(), false, std::numeric_limits<int>::max(), parent )
+  : QgsChunkedEntity( map.maxTerrainScreenError(), map.terrainGenerator(), false, std::numeric_limits<int>::max(), std::numeric_limits<long long>::max(), parent )
```

Comment:
Hmm doesn't this make things worse for terrain if there's no limit for memory at all?